### PR TITLE
feat: add new components

### PR DESCRIPTION
**Pull request by ruchiragoswami**
Add new components : checkbox, radio button 

newComponent

### CODE REVIEW

File: 1-documentation/doc-inputs/doc-inputs.html [hunk 1]
```diff
@@ -131,6 +131,84 @@ <h2> Input with error </h2>
...
+                style="width: 677px; height: 429px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+        </div>
+
+        <h2> Radio Button 1 </h2>
+        <div class="product-filters">
+            <h3> Rating </h3>
+            <section>
+                <label for="four-stars"> <input type="radio" id="four-stars" name="four-stars" value="four-stars"> 4
```
Comment (RakshaPawar108) [inline 1-documentation/doc-inputs/doc-inputs.html]: As you can see in the attached screenshot, I am able to select all radio buttons at once whereas the purpose of radio button is to be able to select only one option at a time. You can fix this issue by giving same `name`  attribute to all the radio buttons among which you want the user to select only one option. Here you can give the `name` as `star-rating` and that will do the job.
Screenshot (130)

File: 1-documentation/doc-inputs/doc-inputs.html [hunk 2]
```diff
@@ -131,6 +131,84 @@ <h2> Input with error </h2>
...
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+        </div>
+
+
+        <h2> Radio Button 2 </h2>
+        <div class="product-filters">
+            <h3> Sort by </h3>
+            <section>
+                <label for="low-to-high"> <input type="radio" id="low-to-high" name="low-to-high" value="low-to-high">
```
Comment (RakshaPawar108) [inline 1-documentation/doc-inputs/doc-inputs.html]: Similarly here you can give `name` attribute as `sort-by-price` to allow user to select only one radio button among the 2.

**Review by RakshaPawar108 — CHANGES_REQUESTED** [review]

**Review by mohammed-sas — APPROVED** [review]
other than suggested by raksha your pr is good to merge

**Review by RakshaPawar108 — APPROVED** [review]
Works fine Ruchi. Code looks good to merge. Great work!

**Pull request merged**